appinventor/appengine/build.xml

```diff
@@ -379,6 +379,7 @@
         <pathelement location="${lib.dir}/guava/guava-18.0.jar" />
         <pathelement location="${lib.dir}/gwt_dragdrop/gwt-dnd-3.2.3.jar" />
         <pathelement location="${lib.dir}/gwt_incubator/gwt-incubator-20101117-r1766.jar" />
+        <pathelement location="${lib.dir}/moxieapps/org.moxieapps.gwt.uploader-1.1.0.jar" />
```

Comment:
I'm concerned that a version of GWT in the future will be incompatible with this code. It is already 4 years old. I would rather not have this functionality, then have it and then have to remove it later.